### PR TITLE
Don't need to Run Samples in Travis CI

### PR DESCRIPTION
**Pull request by iNoles**
## Description
Sample is like show developers what is like to run Fuel.

## Type of change

Check all that apply

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactoring (a change which changes the current internal or external interface)
- [ ] This change requires a documentation update

## How Has This Been Tested?

In case you did not include tests describe why you and how you have verified the
changes, with instructions so we can reproduce. If you have added comprehensive
tests for your changes, you may omit this section.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if necessary
- [x] My changes generate no new compiler warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Inspect the bytecode viewer, including reasoning why

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -32,4 +32,3 @@ include(":fuel-kotlinx-serialization")
 include(":fuel-moshi")
 include(":fuel-test")
 include(":fuel-json")
-include(":sample")
```
Comment (kittinunf) [inline settings.gradle.kts]: [Q] By doing this it means that it won't recognize as a module in IntelliJ? Is this the intended effect?

Comment (SleeplessByte) [reply settings.gradle.kts]: I don't think so

Comment (iNoles) [reply settings.gradle.kts]: Intellij IDEA able to search everything under new source set. It should detect sample.

**Comment by SleeplessByte** [comment]
It adds it back to settings it seems

**Comment by iNoles** [comment]
Closed this for better PR.

**Pull request closed without merging**